cypress/support/step_definition/addons/goToAddonsSteps.spec.js

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
Why keep the feature (see above), when it actually can't test anything since the steps are commented out?